ArkhamOverlay/Pages/LocalImages/LocalImagesController.cs

```diff
@@ -0,0 +1,204 @@
+﻿using ArkhamOverlay.Data;
+using ArkhamOverlay.Services;
+using Microsoft.WindowsAPICodePack.Dialogs;
+using Microsoft.WindowsAPICodePack.Shell;
+using Newtonsoft.Json;
+using PageController;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Windows;
+using System.Windows.Media.Imaging;
+
+namespace ArkhamOverlay.Pages.LocalImages {
+    public class LocalImagesController : Controller<LocalImagesView, LocalImagesViewModel> {
+        private readonly AppData _appData;
+        private readonly LoggingService _logger;
+
+        public LocalImagesController(AppData appData, LoggingService logger) {
+            _appData = appData;
+            _logger = logger;
+            ViewModel.Configuration = appData.Configuration;
+
+            LoadPacks();
+        }
+
+        internal void ShowView() {
+            View.ShowDialog();
+        }
+
+        private void LoadPacks() {
+            if (!Directory.Exists(_appData.Configuration.LocalImagesDirectory)) {
+                _logger.LogMessage($"Directory '{_appData.Configuration.LocalImagesDirectory}' not found.");
+
+            }
+
+            var packs = new List<LocalPack>();
+            try {
+                _logger.LogMessage($"Loading packs from {_appData.Configuration.LocalImagesDirectory}.");
+                foreach (var directory in Directory.GetDirectories(_appData.Configuration.LocalImagesDirectory)) {
+                    packs.Add(LoadPack(directory));                    
+                }
+                ViewModel.Packs = packs;
+            } catch (Exception e) {
+                _logger.LogError(e.Message);
```

Comment:
Use `LogException`. We can also add an overload that only takes in the exception if we don't want to write a message every time.